### PR TITLE
Remove cli from main repo

### PR DESCRIPTION
**Pull request by cartyc**
Due to the lack of maintenance of the `arete` cli, I am proposing it's removal from the toolkit for the time being.

It would be great to bring it back in at a future date but for the time being maintaining the patches is not being done and I would like to avoid distributing code with vulns in it by accident.

### CODE REVIEW

File: solutions/core-landing-zone/mgmt-project/services.yaml [hunk 1]
```diff
@@ -21,9 +21,10 @@ metadata:
   annotations:
     cnrm.cloud.google.com/deletion-policy: "abandon"
     cnrm.cloud.google.com/disable-dependent-services: "false"
-    cnrm.cloud.google.com/project-id: management-project-id # kpt-set: ${management-project-id}
 spec:
   resourceID: cloudbilling.googleapis.com
+  projectRef:
```
[the remark this answers is not in the document]
Comment (cartyc) [reply solutions/core-landing-zone/mgmt-project/services.yaml:26]: Yup, will fix that. Apparently I didn't reset on main when I made the branch.

[the remark this answers is not in the document]
Comment (cartyc) [reply solutions/core-landing-zone/mgmt-project/services.yaml]: rebased and pushed.

**Review by fmichaelobrien — APPROVED** [review]
/lgtm
Will resubmit after linting

**Review by shaunmitchellve — APPROVED** [review]
I have issues with removing the CLI from this repo as it's not being used. I will be maintaining the CLI in my own repo going forward.

**Comment by cartyc** [comment]
Removed CodeQL ci step as we no longer have Go code in the repo.

**Review by fmichaelobrien — APPROVED** [review]
/lgtm

**Comment by cartyc** [comment]
Adresses issue #666

**Review by shaunmitchellve — APPROVED** [review]
LGTM

**Pull request merged**